dist/src/conf/dogma.json

```diff
@@ -28,14 +28,18 @@
   },
   "webAppEnabled": true,
   "webAppTitle": null,
-  "mirroringEnabled": null,
+  "mirroringEnabled": true,
   "numMirroringThreads": null,
   "maxNumFilesPerMirror": null,
   "maxNumBytesPerMirror": null,
   "replication": {
     "method": "NONE"
   },
-  "csrfTokenRequiredForThrift": null,
+  "writeQuotaPerRepository": {
+    "requestQuota": 5,
+    "timeWindowSeconds": 1
+  },
```

Comment:
I don't think this `writeQuotaPerRepository` is a sensible default.
Instead, what do you think of removing the settings in https://github.com/line/centraldogma/blob/a5eb4129dc13a6001c93ac02f70a621a51aa978a/site/src/sphinx/setup-configuration.rst#L50-L53
; set null?

Reply:
Hi @ikhoon, from my view as a user, I think it is better to keep the documentation because it is easy to understand to set specific quotas. I should update this config to null.
Is it good?

Reply:
@ikhoon Any thought? 🤔

Reply:
We haven't fully verified the feature yet. Let's keep it null.

Reply:
Then, @thachlp Please set this to null as @ikhoon  suggested. 🙇 
https://github.com/line/centraldogma/blob/a5eb4129dc13a6001c93ac02f70a621a51aa978a/site/src/sphinx/setup-configuration.rst?plain=1#L50-L53